src/components/ReportingDetail/ReportingDetailDataGrid.tsx

```diff
@@ -53,13 +56,11 @@ const ReportingDetailDataGrid = () => {
       sortable: false,
       align: 'right',
       headerAlign: 'right',
-      renderCell: (params: GridRenderCellParams<ReportingDetailDataRow>) => (
+      renderCell: () => (
         <IconButton
           color="primary"
           size="small"
-          onClick={() => {
-            console.log(`Payment detail: ${params.row.id}`);
-          }}
+          onClick={() => navigate(PageRoutes.REPORTING_PAYMENT_DETAIL)}
```

Comment:
I think we should use a react-router or mui Link component instead of an IconButton, for a11y puroposes